community/modules/scheduler/schedmd-slurm-gcp-v6-controller/partition.tf

```diff
@@ -93,6 +93,8 @@ locals {
     maintenance_interval             = ns.maintenance_interval
     instance_properties_json         = ns.instance_properties_json
     enable_placement                 = ns.enable_placement
+    placement_max_distance           = ns.placement_max_distance
+    machine_type                     = ns.machine_type
```

Comment:
This information should be taken from template, please don't add it, since it's an avenue for inconsistency.

Reply:
Removed `machine_type`